R-package/README.md

```diff
@@ -221,6 +251,73 @@ At build time, `configure.win` will be run and used to create a file `Makevars.w
 1. Edit `configure.win` directly
 2. Edit `src/Makevars.win.in`
 
+### Build Precompiled Binaries of the CRAN Package
+
+This section is mainly for maintainers. As long as the R package is not available on CRAN (which will build precompiled binaries automatically) you may want to build precompiled versions of the R package manually, since these will be easier for users to install.
+
+For more details, see ["Writing R Extensions"](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Building-binary-packages).
+
+Packages built like this will only work for the minor version of R used to build them. They may or may not work across different versions of operating systems
+
+**Mac**
+
+Binary produced: `lightgbm_${VERSION}-r40-macos.tgz`
+
+```shell
+LGB_VERSION="3.0.0-1"
+sh build-cran-package.sh
+R CMD INSTALL --build lightgbm_${LGB_VERSION}.tar.gz
+mv \
+    lightgbm_${LGB_VERSION}.tgz \
+    lightgbm_${LGB_VERSION}-r40-macos.tgz
+```
+
+**Linux**
+
+Binary produced: `lightgbm_${VERSION}-r40-linux.tar.gz`
+
+You can access a Linux system that has R and its build toolchain installed with the `rocker` Docker images.
+
+```shell
+R_VERSION=4.0.2
+
+docker run \
+    -v $(pwd):/opt/LightGBM \
+    -it rocker/verse:${R_VERSION} \
+        /bin/bash
+```
+
+From inside that container, the commands to create a precompiled binary are very similar
+
+```shell
+cd /opt/LightGBM
+LGB_VERSION="3.0.0-1"
+sh build-cran-package.sh
+R CMD INSTALL --build lightgbm_${LGB_VERSION}.tar.gz
+mv \
+    lightgbm_${LGB_VERSION}_R_*-linux-gnu.tar.gz \
+    lightgbm_${LGB_VERSION}-r40-linux.tgz
+```
+
+Exit the container, and the binary package should still be there on the host system.
+
+```shell
+exit
+```
+
+**Windows**
+
+Binary produced: `lightgbm_${VERSION}.zip`
```

Comment:
At release we have
```
lightgbm-${VERSION}.zip
        ^
```

![image](https://user-images.githubusercontent.com/25141164/89718606-87028f80-d9c8-11ea-9123-5be2230bd487.png)


Was it a typo?